src/lib/components/manualOnOffCluster.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+    import { changePlugState, getPlugState } from '$lib/backend-api';
+    import type { PowerState } from '$lib/response-types';
+    import { onMount } from 'svelte';
+    import { Lightbulb, LightbulbOff } from '@lucide/svelte';
+
+    export let background_styles: string | null = null;
+    export let indicator: boolean = true;
+
+    let loading = true;
+    let currentPowerState: boolean = false;
+    let onColor = '';
+    let offColor = '';
```

Comment:
Das kann einfach so direkt gesetzt werden.

```suggestion
    const onColor = 'var(--color-yellow-500)';
    const offColor = 'var(--color-surface-300)';
```

Reply:
Bruh okay, dann wurde ich massiv hinters licht geführt :joy: